internal/telemetry/cluster_test.go

```diff
@@ -66,6 +65,21 @@ func TestClusterIDErrorsOnNotExistingService(t *testing.T) {
 	}
 }
 
+func TestK8sVersionRetrievesClusterVersion(t *testing.T) {
+	t.Parallel()
+
+	c := newTestCollectorForClusterWithNodes(t, node1)
+	got, err := c.K8sVersion()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	want := "v1.29.2"
```

Comment:
qq on this test: We will have to update this string `v1.29.2` again? If so could we test that this matches a semver pattern instead?

Reply:
Ignore this ^ I read below
`td := telemetry.Data{
		K8sVersion: "v1.29.2",
		Arch:       runtime.GOARCH,
	}`